webui/react/src/types.ts

```diff
@@ -406,18 +398,32 @@ export interface Step extends WorkloadGroup, StartEndTimes {
   training: MetricsWorkload;
 }
 
-export interface Metrics {
-  numInputs?: number;
-  validationMetrics?: Record<string, number>;
+export interface Metrics extends Api.V1Metrics {
+  // these two fields are present in the protos
+  // as a struct and list of structs, respectively
+  // here, we are being a bit more precise
```

Comment:
is this assumption valid? pending confirmation from @mpkouznetsov

Reply:
Yes.

Reply:
my default would be to wait until it appears in another structure, but i am not opposed.